mne/viz/backends/_pyvista.py

```diff
@@ -668,18 +668,13 @@ def _update_picking_callback(self,
                                  on_button_press,
                                  on_button_release,
                                  on_pick):
-        self.plotter.iren.AddObserver(
-            vtk.vtkCommand.RenderEvent,
-            on_mouse_move
-        )
-        self.plotter.iren.AddObserver(
-            vtk.vtkCommand.LeftButtonPressEvent,
-            on_button_press
-        )
-        self.plotter.iren.AddObserver(
-            vtk.vtkCommand.EndInteractionEvent,
-            on_button_release
-        )
+        try:
+            add_obs = self.plotter.iren.AddObserver
+        except AttributeError:
+            add_obs = self.plotter.iren.add_observer
```

Comment:
I think it would be good to add a comment why we do this / which line corresponds to which version

Reply:
Very good advice, thank you